projects/1.20/assets/bovines-and-buttercups/bovinesandbuttercups/lang/zh_cn.json

```
@@ -0,0 +1,62 @@
+{
+    "advancements.husbandry.bovinesandbuttercups.prevent_an_effect.title": "状态免除",
+    "advancements.husbandry.bovinesandbuttercups.prevent_an_effect.description": "使用花蜜避免状态效果的影响。",
+    "advancements.husbandry.bovinesandbuttercups.lock_an_effect.title": "状态锁定",
+    "advancements.husbandry.bovinesandbuttercups.lock_an_effect.description": "使用花蜜锁定状态效果的持续时间。",
+    "advancements.husbandry.bovinesandbuttercups.mutate_a_moobloom.title": "斑斓银河",
+    "advancements.husbandry.bovinesandbuttercups.mutate_a_moobloom.description": "繁殖出变异的哞花子代。",
+    
+    "block.bovinesandbuttercups.buttercup": "毛茛",
+    "block.bovinesandbuttercups.pink_daisy": "粉色雏菊",

```

```suggestion
    "block.bovinesandbuttercups.pink_daisy": "粉红色雏菊",
```
原版十六色，参见`粉红色郁金香`/`Pink Tulip`